### PR TITLE
[IMP] account, website_sale: improve docstrings / notes

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Purpose is to cleanup some remaining notes in code to better know some cleaning
is long awaited, planned but still not in WIP.

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
@qdp-odoo Pong !

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
@robodoo r-

File: addons/account/models/mail_thread.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def _message_post_process_attachments(self, attachments, attachment_ids, message
         """ This method extension ensures that, when using the "Send & Print" feature, if the user
         adds an attachment, the latter will be linked to the record.
 
-        # TDE NOTE: clean that brol
+        # SMTeamNote: move to model-based method (see task-2792146)
```
Comment (qdp-odoo) [inline addons/account/models/mail_thread.py:13]:
```suggestion
        #  task-2792146 aims to move to model-based method
```

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**